charts/rancher-vsphere-csi/values.yaml

```diff
@@ -198,6 +204,9 @@ versionOverrides:
           csiProvisioner:
             repository: rancher/mirrored-sig-storage-csi-provisioner
             tag: v3.4.0
+          csiSnapshotter:
+            repository: rancher/mirrored-sig-storage-csi-snapshotter
+            tag: v6.2.1
```

Comment:
v6.2.1 is used in the upstream `vsphere-csi-driver` release v3.0.2
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v3.0.2/manifests/vanilla/vsphere-csi-driver.yaml#L397